src/aks-preview/azext_aks_preview/managed_cluster_decorator.py

```diff
@@ -2945,6 +2945,7 @@ def get_special_parameter_default_value_pairs_list(self) -> List[Tuple[Any, Any]
             (self.context.get_cluster_autoscaler_profile(), None),
             (self.context.get_api_server_authorized_ip_ranges(), None),
             (self.context.get_nodepool_labels(), None),
+            (self.context.get_nodepool_taints(), None),
```

Comment:
You'll need to add a getter function to the mc decorator context to get the value from agent pool decorator context.

See https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/azure/cli/command_modules/acs/managed_cluster_decorator.py#L778